bounties/packagist/ffmpeg-web-gui/1/vulnerability.json

```diff
@@ -0,0 +1,48 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2020-07-26",
+  "AffectedVersionRange": "*",
+  "Summary": "Remote Code Execution",
+  "Author": {
+    "Username": "Mik317",
+    "Name": "Michele Romano"
+  },
+  "Package": {
+    "Registry": "packagist",
+    "Name": "ffmpeg-web-gui",
+    "URL": "https://github.com/swooningfish/ffmpeg-web-gui"
+  },
+  "CWEs": [
+    {
+      "ID": "78",
+      "Description": "Arbitrary Command Injection"
```

Comment:
I think we should add more content to the description, especially for new [huntr.dev](https://huntr.dev) users.

_As a sample:_
Arbitrary Command Injection due to negligent use of `exec()` functions.